plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1186,6 +1186,15 @@ public void testSchemaEvolution()
         dropTable("test_schema_evolution_drop_middle");
     }
 
+    @Test
+    public void testDropPartitionColumn()
+    {
+        String tableName = "test_drop_partition_column_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (col0 INTEGER, col1 INTEGER, col2 INTEGER) WITH (partitioning = Array['col2'])");
```

Comment:
We should avoid meaningless number suffix:
```suggestion
        assertUpdate("CREATE TABLE " + tableName + " (col INTEGER, part INTEGER) WITH (partitioning = ARRAY['part'])");
```